mittab/templates/pairing/manage_room_tags.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base/__normal.html" %}
+
+{% block title %} Manage Room Tags {% endblock title %}
+{% block banner %} Manage Room Tags {% endblock banner %}
+
+
+{% block content %}
+
+<div class="col-md-6 ">
+    <h5>Current Room Tags</h5>
+    <table class="table rounded shadow-sm table-bordered table-resonsive">
+      <thead><th>Tag</th><th>Priority</th><th>Delete</th></thead>
+    
+      {% for tag in room_tags %}
+      <tr>
+        <td >
+            <h5>
+            <a href='/room-tag/{{ tag.id }}/' class="badge badge-{{ tag.color }}" >
+            {{ tag.tag }}
```

Comment:
nit: `<a>` tag should be indented 1 level deeper, and `tag.tag` should be 2 levels deeper